### ISSUE TITLE
PrintSetup setLandscape

### ISSUE THREAD

**Issue by franktudor** [cfsimplicity/lucee-spreadsheet#99]
I have the need to turn a few worksheets inside a client's spreadsheet, to landscape. I found in the POI this setLandscape function that takes the worksheet name and true/false for set landscape. Can you add a function where I can call this if needed. I tried to set it up but it was getting messy and figured you could integrate better. I am also using xlsx not xls, but it looks like it can be set for both types...thanks!

**Comment by cfsimplicity** [comment]
Hope this does what you need, Frank:

[setSheetPrintOrientation](https://github.com/cfsimplicity/lucee-spreadsheet/wiki/setSheetPrintOrientation)

**Comment by franktudor** [comment]
**Works with perfect with sheetName with xlsx:**

`spreadsheet.setSheetPrintOrientation( workbook=workbook, mode="landscape", sheetName="Methods and Approach" );`

The sheetNumber parameter didn't work for me.  Maybe remove that extra code (I understand the motivation for it), Unless it 'does' work for xls (I did not test).

**Comment by cfsimplicity** [comment]
Strange. I had only written tests for xls but I've added them for xlsx too now and they pass. Have also tested by eye using xls and xlsx in both Excel Viewer and OpenOffice and the print preview seems to be right.  This is the syntax I used (both are valid):
```
spreadsheet.setSheetPrintOrientation( workbook=workbook, mode="landscape", sheetNumber=2 );
spreadsheet.setSheetPrintOrientation( workbook, "landscape", "", 2 );
```
I think it's worth having the sheetNumber option in case names aren't known and to keep things consistent (the library generally allows specifying sheets by name or number).
